app/renderer/components/preferences/helpfulHints.js

```diff
@@ -40,11 +40,15 @@ const styles = StyleSheet.create({
   },
 
   helpfulHints: {
-    width: '100%',
-    height: '100%',
     cursor: 'default',
-    padding: '30px 0',
-    marginTop: '20px'
+    padding: '20px 0 0',
+    overflowY: 'auto',
+
+    '@media (min-height: 620px)': {
```

Comment:
if we have enough display height then put hints at screen's bottom.

Ideally we should do it on navigation itself but then we lost the active indicator (arrow)